Makefile

```diff
@@ -122,8 +122,8 @@ vet:
 # Evergreen specific targets
 .PHONY: evg-test
 evg-test:
-	for TEST in $(TEST_PKGS) ; do \
-		LD_LIBRARY_PATH=$(LD_LIBRARY_PATH) PKG_CONFIG_PATH=$(PKG_CONFIG_PATH) go test $(BUILD_TAGS) -v -timeout $(TEST_TIMEOUT)s $$TEST >> test.suite ; \
+	for TEST in $(TEST_PKGS); do \
+		go test -exec "env PKG_CONFIG_PATH=$(PKG_CONFIG_PATH) LD_LIBRARY_PATH=$(LD_LIBRARY_PATH)" $(BUILD_TAGS) -v -timeout $(TEST_TIMEOUT)s $$TEST >> test.suite ; \
```

Comment:
Are there necessary documentation changes for users running tests on macOS?

Reply:
We recommend users follow  https://github.com/mongodb/libmongocrypt#installing-libmongocrypt-on-macos for installing libmongocrypt on macos and those instructions install into a standard location, so setting `LD_LIBRARY_PATH` and `PKG_CONFIG_PATH` shouldn't be necessary. This has only been an issue for us in Evergreen because we can't reliably install things into the standard system locations. Given that, I don't think documentation updates are needed.